spec.html

```diff
@@ -47804,8 +48373,8 @@ <h2>Syntax</h2>
       <emu-grammar type="definition">
         Term[UnicodeMode, N] ::
           [+UnicodeMode] Assertion[+UnicodeMode, ?N]
-          [+UnicodeMode] Atom[+UnicodeMode, ?N] Quantifier
-          [+UnicodeMode] Atom[+UnicodeMode, ?N]
+          [+UnicodeMode] Atom[+UnicodeMode, ?UnicodeSetsMode, ?N] Quantifier
```

Comment:
A production can't have `?UnicodeSetsMode` in the RHS when it doesn't have `UnicodeSetsMode` in the LHS.

You'll need to add `UnicodeSetsMode` to `Term`, `Assertion`, `QuantifiableAssertion`, and `ExtendedAtom`.

Reply:
PTAL: https://github.com/mathiasbynens/ecma262/pull/25